packages/dynamic-links/lib/modular/index.d.ts

```diff
@@ -0,0 +1,141 @@
+import { ReactNativeFirebase } from '@react-native-firebase/app';
+import { FirebaseDynamicLinksTypes } from '..';
+
+type DynamicLinks = FirebaseDynamicLinksTypes.Module;
+
+export declare const ShortLinkType = FirebaseDynamicLinksTypes.ShortLinkType;
+
+/**
+ * Returns an instance of DynamicLinks.
+ *
+ * #### Example
+ * ```js
+ * const dynamicLinks = getDynamicLinks();
+ * ```
+ */
+export declare function getDynamicLinks(app?: ReactNativeFirebase.FirebaseApp): DynamicLinks;
```

Comment:
I don't think our typings should be react-native-firebase specific?

```suggestion
export declare function getDynamicLinks(app?: FirebaseApp): DynamicLinks;
```

Reply:
other modules solve this with an explicit import above, but it reduces to the same statement and this is more concise. I suppose it is unavoidable